src/org/thoughtcrime/securesms/ConversationTitleView.java

```diff
@@ -107,9 +107,6 @@ else if( dcChat.isDeviceTalk() ) {
       else {
         DcContact dcContact = dcContext.getContact(chatContacts[0]);
         subtitleStr = dcContact.getAddr();
-        if (dcContact.isVerified()) {
-          imgRight = R.drawable.ic_verified;
-        }
```

Comment:
I'd actually say that this was a bug: We showed a verified icon whenever we had a verified key of this contact, even if the messages in the chat were not verified with this key.

Reply:
yes, it was wrong